bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/TlsTrustManagerProvider.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.io.net.http;
+
+import javax.net.ssl.X509ExtendedTrustManager;
```

Comment:
@kaikreuzer @maggu2810 Is this `javax` a problem :'(

Reply:
At least also the following bundles contains a package import of `javax.net.ssl`.

* org.eclipse.jetty.io
* org.eclipse.jetty.jmx
* org.eclipse.jetty.server
* org.eclipse.jetty.util

The Jetty Util bundle is used by most of the Jetty bundles, so as long as Jetty needs it, it should not be a problem for us (IMHO).

But please don't miss do change the manifest, too (import package `javax.net.ssl`).

Reply:
Pff, I was afraid it was all in vain :-) Thanks!!!

Reply:
I do not understand. Is `javax` a bad namespace? All the ssl goodies are in there.

Reply:
I mistakenly assumed that javax was completely not a part of compact profile, but indeed some of the Javax packages are part of the compact profile 👍